pyro/poutine/trace.py

```diff
@@ -194,6 +281,14 @@ def observation_nodes(self):
                 if node["type"] == "sample" and
                 node["is_observed"]]
 
+    @property
+    def param_nodes(self):
+        """
+        Gets a list of names of param sites
```

Comment:
can you write this more sphinx-formally ie `:return: list of names of param sites`